### PR TITLE
[FAB-17632] Exclude unnecessary code from ecdsa.go 

### PR DESCRIPTION
**Pull request by vadiminshakov**
Fixed GenSKI() for clean code.

### CODE REVIEW

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go [hunk 1]
```diff
@@ -35,26 +36,32 @@ type SubjectKeyIdentifier struct {
...
 
 	// Marshall the public key
 	raw := elliptic.Marshal(k.PubKey.Curve, k.PubKey.X, k.PubKey.Y)
 
 	// Hash it
 	hash := sha256.New()
-	hash.Write(raw)
+	_, err := hash.Write(raw)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Failed to write hash, error: %s", err))
```
Comment (kopaygorodsky) [inline scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go]: why new error? use Wrapf

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go [hunk 2]
```diff
@@ -35,26 +36,32 @@ type SubjectKeyIdentifier struct {
...
+	}
 	k.SKI.Sha256Bytes = hash.Sum(nil)
 	k.SKI.Sha256 = hex.EncodeToString(k.SKI.Sha256Bytes)
 
 	hash = sha1.New()
-	hash.Write(raw)
+	_, err = hash.Write(raw)
+	if err != nil {
+		return errors.New(fmt.Sprintf("Failed to write hash, error: %s", err))
```
Comment (kopaygorodsky) [inline scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go]: errors.Wrapf

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/pkcs11.go [hunk 3]
```diff
@@ -236,7 +236,11 @@ func (p11w *Pkcs11Wrapper) ImportECKey(ec EcdsaKey) (err error) {
 		return
 	}
 
-	ec.GenSKI()
+	err = ec.GenSKI()
+	if err != nil {
+		err = errors.New(fmt.Sprintf("failed to generate SKI, error: %s", err))
```
Comment (kopaygorodsky) [inline scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/pkcs11.go]: errors.Wrapf

**Review by kopaygorodsky — CHANGES_REQUESTED** [review]

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go (hunk 1 above)
Comment (VadimInshakov) [reply scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go]: Ok!

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go (hunk 2 above)
Comment (VadimInshakov) [reply scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/ecdsa.go]: Ok!

File: scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/pkcs11.go (hunk 3 above)
Comment (VadimInshakov) [reply scripts/_go/src/pkcs11helper/pkg/pkcs11wrapper/pkcs11.go]: Ok!

**Comment by troyronda** [comment]
@VadimInshakov Thanks for your contribution. Please update your commit and PR with DCO sign-off.

**Comment by VadimInshakov** [comment]
@troyronda ok, done.

**Comment by VadimInshakov** [comment]
@troyronda is there any updates?

**Review by troyronda — APPROVED** [review]

**Pull request merged**